zulip/integrations/slack/slackdata2zulipdata.py

```diff
@@ -139,20 +141,32 @@ def channels2zerver_stream(slack_dir, realm_id, added_users):
             id=stream_id_count)
         zerver_stream.append(stream)
         added_channels[stream['name']] = stream_id_count
+       
+        # construct the recipient object and append it zerver_recipient
+        recipient = dict(
+             type_id=stream_id_count,
+             id=stream_id_count,
+             type=2)
+        zerver_recipient.append(recipient)
+        # TODO add recipients for private messages and huddles 
 
         # construct the subscription object and append it to zerver_subscription
         for member in channel['members']:
             sub = dict(
-                recipient=added_users[member],
+                recipient=stream_id_count,
                 notifications=False,
                 color="#c2c2c2",
                 desktop_notifications=True,
                 pin_to_top=False,
                 in_home_view=True,
                 active=True,
                 user_profile=added_users[member],
-                id=stream_id_count)  # TODO is this the correct interpretation?
+                id=subscription_id_count)  # TODO is this the correct interpretation?
```

Comment:
Can this TODO be removed? Maybe provide the documentation / code link as a proof?